src/Store/store.ts

```diff
@@ -1,10 +1,12 @@
-import { createStore } from 'redux'
+import { createStore } from 'redux';
 
 const initialState = {
```

Comment:
Let's have a clear interface (or type) for the whole store. I think not only weatherData should be in the redux store, but also other details like the current selected temperature unit.